standard_forms/purchase-order/latest/src/template.html

```diff
@@ -53,8 +53,10 @@
       currency")}}:
     </td>
     <td>
+      {% if data.purchaseOrder.orderTotalAfterDiscount and
```

Comment:
Why do we need logo to display the discount? Given this is not a real world issue I think it would be fine to close?